week01/insert_delete_getRandom.cpp

```diff
@@ -0,0 +1,50 @@
+//TC: O(1)
+//SC: O(2N)
+class RandomizedSet {
+public:
+    vector<int>rSet; ///randmize Set
+    unordered_map<int,int>idxMap;
+
+    RandomizedSet() {
+        rSet.clear();
+    }
+    
+    bool insert(int val) {
+        if(idxMap.count(val))return false; ///already present
```

Comment:
```suggestion
        if(idxMap.count(val))return false;
```

Please do not put unnecessary comments. It is already clear what you are doing here.

Fix it for other cases as well.